Resources/doc/reference/form_field_definition.rst

```diff
@@ -227,16 +227,23 @@ You can easily add a new ``galleryHasMedias`` row by defining one of these optio
                 ->add('enabled')
                 ->add('name')
                 ->add('defaultFormat')
-                ->add('galleryHasMedias', 'sonata_type_collection', array(), array(
-                    'edit' => 'inline',
-                    'inline' => 'table',
-                    'sortable' => 'position',
-                    'limit' => 3
+                ->add('galleryHasMedias', 'sonata_type_collection', array(
+                        'by_reference' => false
+                    ), 
+                    array(
+                        'edit' => 'inline',
+                        'inline' => 'table',
+                        'sortable' => 'position',
+                        'limit' => 3
                 ))
             ;
         }
     }
+    
+.. note:: 
 
+    You have to define ``setMedias`` method into your ``Gallery`` class and manually attach each ``media`` to the current ``gallery`` and define the media's relation into ``gallery`` as cascade persist.
```

Comment:
- "the" `setMedias` method
- define cascading persistence for the relationship from media to gallery